fsw/src/ci_lab_app.c

```diff
@@ -203,582 +217,180 @@ void CI_TaskInit(void)
+    CI_LAB_Global.HkBuffer.HkTlm.Payload.CommandCounter++;
 
-        case CI_STOP_PDU_CAPTURE_CC:
-            CI_StopPDUCaptureCmd(CIMsgPtr);
-            break;
+    CFE_EVS_SendEvent(CI_LAB_COMMANDNOP_INF_EID, CFE_EVS_EventType_INFORMATION, "CI: NOOP command");
 
-        /* default case already found during FC vs length test */
-        default:
-            break;
-    }
+    return CFE_SUCCESS;
+}
 
-    return;
 
-} /* End of CI_ProcessGroundCommand() */
+/* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * */
+/*  Name:  CI_LAB_ResetCounters                                                */
+/*                                                                             */
+/*  Purpose:                                                                   */
+/*     Handle ResetCounters command packets                                    */
+/*                                                                             */
+/* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * */
+int32 CI_LAB_ResetCounters(const CI_LAB_ResetCounters_t *data)
+{
+    CFE_EVS_SendEvent(CI_LAB_COMMANDRST_INF_EID, CFE_EVS_EventType_INFORMATION, "CI: RESET command");
+    CI_LAB_ResetCounters_Internal();
+    return CFE_SUCCESS;
+}
 
 /* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * **/
-/*  Name:  CI_ReportHousekeeping                                              */
+/*  Name:  CI_LAB_ReportHousekeeping                                          */
 /*                                                                            */
 /*  Purpose:                                                                  */
 /*         This function is triggered in response to a task telemetry request */
 /*         from the housekeeping task. This function will gather the CI task  */
 /*         telemetry, packetize it and send it to the housekeeping task via   */
 /*         the software bus                                                   */
 /* * * * * * * * * * * * * * * * * * * * * * * *  * * * * * * *  * *  * * * * */
-void CI_ReportHousekeeping(void)
+int32 CI_LAB_ReportHousekeeping(const CCSDS_CommandPacket_t *data)
 {
-    CI_HkTelemetryPkt.SocketConnected = CI_SocketConnected;
-    CFE_SB_TimeStampMsg((CFE_SB_Msg_t *)&CI_HkTelemetryPkt);
-    CFE_SB_SendMsg((CFE_SB_Msg_t *)&CI_HkTelemetryPkt);
-    return;
+    CI_LAB_Global.HkBuffer.HkTlm.Payload.SocketConnected = CI_LAB_Global.SocketConnected;
+    CFE_SB_TimeStampMsg(&CI_LAB_Global.HkBuffer.MsgHdr);
+    CFE_SB_SendMsg(&CI_LAB_Global.HkBuffer.MsgHdr);
+    return CFE_SUCCESS;
 
-} /* End of CI_ReportHousekeeping() */
+} /* End of CI_LAB_ReportHousekeeping() */
 
 /* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * **/
-/*  Name:  CI_ResetCounters                                                   */
+/*  Name:  CI_LAB_ResetCounters                                               */
```

Comment:
CI_LAB_ResetCounters_Internal

Reply:
@skliper are you requesting this name be changed?

Reply:
correct.